### PR TITLE
Remove test kitchen

### PR DESCRIPTION
**Pull request by adborden**
Part of https://github.com/GSA/datagov-deploy/issues/581

Remove test kitchen which isn't really used.

### CODE REVIEW

**Comment by jbrown-xentity** [comment]
@adborden was Snyk using test kitchen for testing, is that why the check failed? I'm not completely comfortable with merging until that issue (either updating Snyk per documentation above or verification that system works) is resolved.

**Comment by adborden** [comment]
Right, the Snyk check is failing because the Gemfile.lock was removed and you can safely ignore the check.

FYI, in general, this shouldn't hold up the a review. Any CI checks that should block a PR are set as "required" in GitHub. So you can safely approve the PR even if tests are failing. It's on the Author to resolve any required failing checks and get it merged.

**Review by jbrown-xentity — APPROVED** [review]
Looks clean

**Pull request merged**
